modules/common/src/main/ds/DownloadCsv.scala

```diff
@@ -0,0 +1,77 @@
+package notebook.front.widgets
```

Comment:
let's remove `df/DownloadCsv.scala`, and use only native spark csv. no more issues with lib versioning.